tideways/config.php-profiler.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+require_once 'vendor/autoload.php';
+
+$config = array(
+    'save.handler' => \Xhgui\Profiler\Profiler::SAVER_PDO,
+    'save.handler.pdo' => array(
+        'dsn' => 'sqlite:/tmp/xhgui.sqlite3',
+        'user' => null,
+        'pass' => null,
+        'table' => 'results'
+    ),
+    // If defined, use specific profiler
+    // otherwise use any profiler that's found
+    'profiler' => \Xhgui\Profiler\Profiler::PROFILER_TIDEWAYS_XHPROF,
+    // This allows to configure, what profiling data to capture
+    'profiler.flags' => array(
+        \Xhgui\Profiler\ProfilingFlags::CPU,
+        \Xhgui\Profiler\ProfilingFlags::MEMORY,
+        \Xhgui\Profiler\ProfilingFlags::NO_BUILTINS,
+        \Xhgui\Profiler\ProfilingFlags::NO_SPANS,
+    ),
+    // Environment variables to exclude from profiling data
+    'profiler.exclude-env' => array(
+        'APP_DATABASE_PASSWORD',
+        'PATH',
```

Comment:
too much copy paste? do these variables exist in your application? if you want to have an example, keep them commented out?